enterprise/micronaut/src/org/netbeans/modules/micronaut/completion/MicronautDataCompletionTask.java

```diff
@@ -249,9 +250,16 @@ private <T> void resolveFinderMethods(CompilationController cc, TreePath path, S
             for (ExecutableElement method : ElementFilter.methodsIn(entity.getEnclosedElements())) {
                 String methodName = method.getSimpleName().toString();
                 if (methodName.startsWith(GET) && method.getParameters().isEmpty()) {
-                    prop2Types.put(methodName.substring(GET.length()), tu.getTypeName(method.getReturnType()).toString());
+                    methodName = methodName.substring(GET.length());
+                    methodName = methodName.substring(0, 1).toUpperCase() + methodName.substring(1);
+                    prop2Types.put(methodName, tu.getTypeName(method.getReturnType()).toString());
```

Comment:
I'd check the `method.getReturnType()` to be valid / not erroneous, just in case - with unresolved types, generics may work / display bad

Reply:
Fixed.